gateway/src/main/java/org/georchestra/gateway/security/GatewaySecurityConfiguration.java

```diff
@@ -103,6 +124,10 @@ private Stream<ServerHttpSecurityCustomizer> sortedCustomizers(List<ServerHttpSe
         return new ResolveGeorchestraUserGlobalFilter(resolver);
     }
 
+    public @Bean ResolveApacheGeorchestraUserFilter resolveApacheGeorchestraUserFilter() {
+        return new ResolveApacheGeorchestraUserFilter();
+    }
```

Comment:
Apache sounds specific, of course this is what we will actually have with mod_mellon, but we could imagine use this mechanism with whatever we have in front of the gateway (nginx, ...), maybe `ResolveHttpHeadersGeorchestraUserFilter` ?